dags/process_elife_zip_dag.py

```diff
@@ -145,7 +149,7 @@ def convert_tiff_images_in_expanded_bucket_to_jpeg_images(**context) -> None:
 
 def update_tiff_references_to_jpeg_in_article(**context) -> bytes:
     zip_file_name = get_file_name_passed_to_dag_run_conf_file(context)
-    article_name = get_expected_elife_article_name(zip_file_name)
+    article_name = get_article_file_name(zip_file_name)
```

Comment:
you wanted to improve the interface of this function to return the XML content directly?

Reply:
correct. updated.